scripts/build_images/build.py

```diff
@@ -120,6 +123,9 @@ def build_image(
     should_push: bool = False,
     base_image_digest: str | None = None,
     metadata_dir: str | None = None,
+    cache_strategy: CacheStrategy = "none",
+    cache_only: list[str]
+    | None = None,  # Only will use the cache strategy for these images -- all others will be built with cache_strategy="none".
```

Comment:
Nit, the formatting here looks kind of funky?

Reply:
Resolved in [4df34af](https://github.com/PrairieLearn/PrairieLearn/pull/11723/commits/4df34af3fef05e9290594bbc50ce9f064dccbaa6).